src/commands/voice-channel/leave.ts

```
@@ -1,27 +1,36 @@
-      ephemeral: true,
-    });
-
-    return;
-  }
-
-  connection.destroy();
-
-  // cspell:disable-next-line
-  await interaction.reply({ content: 'okay byeee!', ephemeral: true });
-}
+import { getVoiceConnection } from '@discordjs/voice';
```

ESLint

[error] Unable to resolve path to module '@discordjs/voice'.

(import-x/no-unresolved)